<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateWindow</class>
 <widget class="QDialog" name="UpdateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>710</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="logo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>54</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>54</width>
        <height>54</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../../resources/resources.qrc">:/icons/talkie.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="headerLabel">
        <property name="minimumSize">
         <size>
          <width>614</width>
          <height>85</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>614</width>
          <height>85</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;A new version of Talkie is available!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Talkie {{newVersion}} is now available - you have {{currentVersion}}. Would you like to download it now?&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Release Notes:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="changelog">
        <property name="minimumSize">
         <size>
          <width>614</width>
          <height>133</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>614</width>
          <height>133</height>
         </size>
        </property>
        <property name="lineWrapColumnOrWidth">
         <number>5</number>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>614</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>614</width>
          <height>32</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <widget class="QPushButton" name="installButton">
         <property name="geometry">
          <rect>
           <x>471</x>
           <y>0</y>
           <width>141</width>
           <height>32</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Install Update</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="laterButton">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>0</y>
           <width>141</width>
           <height>32</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Remind Me Later</string>
         </property>
        </widget>
        <widget class="QPushButton" name="skipButton">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>141</width>
           <height>32</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>Skip This Version</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
